.github/workflows/test_full.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       SPLINEPY_GITHUB_ACTIONS_BUILD: True
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11"]
+        python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
```

Comment:
Above in the build workflow we already test and build the package already with `py3.12`.

Reply:
I thought I removed it?

Reply:
Nope, good catch!